### ISSUE TITLE
Make an exe file from blazorwasm (blazorclient-side) project

### ISSUE THREAD

**Issue by alikarooni** [aspnet/AspNetCore#16732]
Is there anyone to answer my question properly.

I am gonna start by a question: which one of the following scenarios I am able to implement?

publishing blazorwasm client-side into exe file( instead of using IIS or apache or Nginx) so I can run it as a windows service

OR

Embedding a dot net core app and blazorwasm client-side (not server side which I know about it already) into the same project

Option-1: One send me the link below to publish blazorwasm project into exe file: https://docs.microsoft.com/en-us/dotnet/core/whats-new/dotnet-core-3-0#single-file-executables which did not work. Because this approach works when your target framework is netcoreapp3.0. But when I run "dotnet new blazorwasm", new blazorwasm project's target is netstandard2.0.

Option-2: Using Microsoft template for creating a new web application by embedding .razor pages: This approach is different than what I want and is not completely a "Single Page Application". For example, the asp.net project takes care of routing not blazorwasm, and .razor pages are kind of partials that load into .cshtml.

To sum up, I want to publish a blazorwasm project (which is a client-side webassembly) and host it into a windows service, just the same as what we can do with dotnetcore application(run in in the background or windows service), instead of publishing them using IIS or Apache.

**Comment by javiercn** [comment]
@alikarooni thanks for contacting us. 

`dotnet new blazorwasm -ho` and publish single file executable for the host is the closest you are going to get. You need an HTTP server to serve the files.

You can run the web app as a windows service. See https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/windows-service?view=aspnetcore-3.0&tabs=visual-studio

**Comment by javiercn** [comment]
Closing this issue as it's a duplicate of https://github.com/aspnet/AspNetCore/issues/16635

**Comment by alikarooni** [comment]
Thanks for keep updating comments. I've read somewhere that blasorwasm is kind of in-process (the same as all .netcore3.0) and that's why you can only create a template using CLI and it is not in the visual studio new project dialog yet, but I think they would implement the part to creating an exe file from blasorwasm client-side without any requirement of combination, or I think they should think about implementing it.
